src/modules/style.ts

```diff
@@ -69,7 +69,7 @@ function applyRemoveStyle(vnode: VNode, rm: () => void): void {
     return;
   }
   if(!reflowForced) {
-    getComputedStyle(document.body).transform;
+    (vnode.elm as any).offsetLeft;
```

Comment:
Thank you for your contribution, @jvanbruegge.

Seems legit to me. I'd like to see #457 fixed in order to conveniently see the test results in all browsers.

Also, #286 would be helpful.

Reply:
For #457 @paldepind would have to write an email to browserstack, should not be more. For OS projects they can activate a free pro account (cyclejs uses the same).

But maybe for now the DOM browserstack test of cyclejs are enough? It's a very niche case that is currently blocking the cyclejs CI

Reply:
We have CI now. And it passed. I could and did handle that. But I don't have permission to merge nor release. @paldepind, you're our only hope.

Reply:
> For #457 @paldepind would have to write an email to browserstack, should not be more. For OS projects they can activate a free pro account (cyclejs uses the same).

That should have been sorted out now :smile: I got an email from Browserstack a couple of weeks ago saying that we now have a "sponsored" account.